spt.c

```diff
@@ -80,16 +80,28 @@ void
 remaining_time(int sigint)
 {
 	char buf[17];
-	if (signal(SIGUSR1, SIG_IGN) != SIG_IGN)
-		signal(SIGUSR1, remaining_time);
+	//if (signal(SIGUSR1, SIG_IGN) != SIG_IGN)
+	//	signal(SIGUSR1, remaining_time);
```

Comment:
I commented this out because I wasn't sure about its utility.
Is it necessary to reset the handler for SIGUSR1 here?